src/wasm/kotlin/tests/arcs/ParticleApiTest.kt

```diff
@@ -0,0 +1,145 @@
+package arcs
```

Comment:
This single file has a lot of classes, consider breaking it up into separate files.

Reply:
Also, please add kdoc for each of the public members.

Reply:
Broke up in 193a6a3 and ce45f24

Reply:
@jasonwyatt Public member functions, fields, or both?

Reply:
They don't have to be on `override` functions/fields, but any newly declared public fields/functions that aren't totally obvious in their meaning should probably be Kdoc'd.

Reply:
All the functions that are public are overriden. All other fields are private.